package.json

```diff
@@ -69,21 +69,21 @@
     "format-fix": "prettier './packages/**/*.ts' --write"
   },
   "devDependencies": {
+    "@eslint/js": "^9.3.0",
```

Comment:
What does this do?

Reply:
this is the part of new setup - https://eslint.org/docs/latest/use/getting-started#manual-set-up

Perhaps we can remove other package from the package.json. Will check